Dockerfile

```diff
@@ -157,6 +157,11 @@ RUN ARCH=$(arch | sed s/aarch64/arm64/) \
     rm requirements.txt; \
     fi
 
+# Extract ScanCode license texts to a directory
+RUN scancode-license-data --path=/opt/scancode-license-data \
```

Comment:
Nit: `scancode-license-data --help` says to use `--path` without `=`, so I'd drop that here.